### DISCUSSION TITLE
Can't jump to video link on discussion.

### DISCUSSION THREAD

**Discussion by yuioto** [xfangfang/wiliwili#208] [General]
I can't use blue link jump to video, and video link no display aid or bid or cid, so I can't fount video.

![图片](https://github.com/xfangfang/wiliwili/assets/68138727/65bf8063-6f67-4a4b-941d-d5e4b7d41217)

我尝试点击蓝色视频标题样式字体，但是我无法跳转到视频详细页，同时由于不显示任何av号等标记，导致我无法只通过wiliwili访问这个视频，除非搜索标题才有可能。

这是否是一个bug？如果不是，那么蓝色样式于播放图标容易误导人们，或许应该修改它，或者为它添加一个跳转功能。

## info

wiliwili v1.1.1.r30.ge486c01 (PS: 应该加一个在关于界面里显示 commit hash 等构建信息的功能方便诊断
Commit ID : e486c01c62430eb4f149ef8169e1c01f932f3d28
build on ArchLinux, use PKGBUILD from https://aur.archlinux.org/wiliwili-git.git
OS : ArchLinux

**Comment by xfangfang** [comment]
@yuioto 不是bug，这个功能还没做。

我的设想是在点赞和评论下方放置一个用于显示可跳转或搜索的列表，如果有需求可以自行实现一下，或者讨论其他可行的方案。
视频跳转会导致应用页面存在无限加载下去的可能，所以我建议可以在当前页面打开视频。
视频搜索也会出现类似的问题，解决方式应该是相似的。

短时间内我可能不会在这个项目上花太多的时间了。
